### PR TITLE
[OSCD] T1552.003: Suspicious history file operations - Linux/macOS

### PR DESCRIPTION
**Pull request by alx1m1k**
For issues [#1011](https://github.com/Neo23x0/sigma/issues/1011) and [#1012](https://github.com/Neo23x0/sigma/issues/1012)

### CODE REVIEW

**Comment by alx1m1k** [comment]
Added both rules to linux folder, because structure for unix-like rules must be changed anyways like proposed in [#1048](https://github.com/Neo23x0/sigma/issues/1048#issuecomment-704608331)

File: rules/linux/macos_susp_histfile_operations.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1552.003/T1552.003.md
+logsource:
+    product: macos
+    category: process_creation
+detection:
+    selection:
+        CommandLine|contains:
+            - '.bash_history'
+            - '.zsh_history'
```
Comment (yugoslavskiy) [inline rules/linux/macos_susp_histfile_operations.yml:21]: Hey mate! You can also add `.sh_history` here (:

Comment (alx1m1k) [reply rules/linux/macos_susp_histfile_operations.yml:21]: Are there a lot of people using ksh in macOS? If so, i'll fix this one

Comment (yugoslavskiy) [reply rules/linux/macos_susp_histfile_operations.yml:21]: Well, actually, the list is bigger. Take a look at what the osquery is watching for in the [`shell_history` table](https://github.com/theopolis/osquery/blob/327a6a3e247efc6420ac6b541cc5ec4349e2e879/osquery/tables/system/posix/shell_history.cpp#L23-L28) (the same for both Linux and macOS).

**Comment by yugoslavskiy** [comment]
Thank you for your contribution, @alx1m1k! Your rules look perfect. 
I am really glad to see you participating this year again!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**